.github/workflows/frontend.yml

```diff
@@ -0,0 +1,96 @@
+name: Frontend
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'release**'
+      - 'feature**'
+      - 'fix**'
+      - 'ci**'
+    tags:
+      - '**'
+    paths:
```

Comment:
Do we need to include `./docs` here?

Reply:
> Do we need to include `docs/` here?

Good catch! Seems we need it for lint-docs-links.